src/algebra/inj_surj.lean

```diff
@@ -0,0 +1,273 @@
+/-
+Copyright (c) 2014 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+
+import algebra.ring
+
```

Comment:
module doc string?

Reply:
Good catch... maybe someone should update the style guidelines to mention module docstrings :upside_down_face: :grimacing: :facepalm:

Reply:
Done